core-services/image-mirroring/openshift-kni/mapping_openshift-kni_quay

```diff
@@ -86,7 +86,8 @@ registry.ci.openshift.org/ocp-kni/4.15:ztp-site-generator quay.io/openshift-kni/
 registry.ci.openshift.org/ocp-kni/4.16:cnf-tests quay.io/openshift-kni/cnf-tests:4.16 quay.io/openshift-kni/cnf-tests:4.16.0 quay.io/openshift-kni/cnf-tests:latest
 registry.ci.openshift.org/ocp-kni/4.16:dpdk quay.io/openshift-kni/dpdk:4.16 quay.io/openshift-kni/dpdk:4.16.0 quay.io/openshift-kni/dpdk:latest
 registry.ci.openshift.org/ocp-kni/4.16:ztp-site-generator quay.io/openshift-kni/ztp-site-generator:4.16 quay.io/openshift-kni/ztp-site-generator:4.16.0
-registry.ci.openshift.org/ocp-kni/4.17:ztp-site-generator quay.io/openshift-kni/ztp-site-generator:4.17 quay.io/openshift-kni/ztp-site-generator:4.17.0 quay.io/openshift-kni/ztp-site-generator:latest
+registry.ci.openshift.org/ocp-kni/4.17:ztp-site-generator quay.io/openshift-kni/ztp-site-generator:4.17 quay.io/openshift-kni/ztp-site-generator:4.17.0
```

Comment:
@rauhersu I think we need to bump the other images in this file as well or at least notify the folks to do this..

Reply:
@yuvalk @rauhersu is going to notify the necessary folks to make the requested updates here once this change merges.. we discussed this.

Reply:
we can consider this comment resolved for the PR merge perspective..